src/main/java/core/basesyntax/behaviour/Colorable.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.behaviour;
+
+import core.basesyntax.type.Color;
+
+public interface Colorable {
+    Color getColor();
+
+    String getStringColor();
```

Comment:
Consider renaming the method 'getStringColor()' to 'getColorAsString()' for better readability.